express/src/routes/manage-account.ts

```diff
@@ -26,17 +26,21 @@ const router = express.Router();
 
 router.get("/account", checkAuthorisation, showAccount);
 
-router.get("/account/list-services", checkAuthorisation, listServices);
+router.get("/services", checkAuthorisation, listServices);
 
-router.get("/add-service-name", checkAuthorisation, showAddServiceForm);
-router.post("/add-service-name", checkAuthorisation, serviceNameValidator, processAddServiceForm);
+router.get("/service-name", checkAuthorisation, showAddServiceForm);
```

Comment:
```suggestion
router.get("/register/create-service", checkAuthorisation, showAddServiceForm);
```
This needs to be decoupled from the account creation.
Move this URL to the `create.ts` file